paymentsheet/src/main/java/com/stripe/android/paymentsheet/model/PaymentSelection.kt

```diff
@@ -388,7 +394,7 @@ internal val PaymentSelection.label: ResolvableString
         is PaymentSelection.ExternalPaymentMethod -> label
         is PaymentSelection.CustomPaymentMethod -> label
         PaymentSelection.GooglePay -> StripeR.string.stripe_google_pay.resolvableString
-        is PaymentSelection.Link -> StripeR.string.stripe_link.resolvableString
+        is PaymentSelection.Link -> label
```

Comment:
Using the nickname as Link label when the `PaymentSelection` has payment details.